pkg/cmd/server/api/validation/master.go

```diff
@@ -253,6 +259,24 @@ func ValidateAuditConfig(config api.AuditConfig, fldPath *field.Path) Validation
 		validationResults.AddErrors(field.Invalid(fldPath.Child("maximumFileSizeMegabytes"), config.MaximumFileSizeMegabytes, "must be greater than or equal to 0"))
 	}
 
+	if len(config.PolicyFile) > 0 {
+		policy, err := auditpolicy.LoadPolicyFromFile(config.PolicyFile)
+		if err != nil {
+			validationResults.AddErrors(field.Invalid(fldPath.Child("policyFile"), config.PolicyFile, err.Error()))
+		}
+		if len(policy.Rules) == 0 {
```

Comment:
Why shouldn't this be treated the same as "no policy file specified"?

Reply:
That's how it's done upstream, I want to be consistent with that behavior.